context/app/static/js/hooks/useSearchHits.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
+import { getAuthHeader } from 'js/helpers/functions';
+
+function useSearchHits(query, elasticsearchEndpoint, nexusToken) {
```

Comment:
Should we file a follow-up issue to use this elsewhere in the codebase?  I don't think this is the PR to do this in but this seems like something we might do often (could be wrong though)?

Reply:
I think we have one place we could use this now, thanks I'll file an issue.